pkg/tfgen/generate.go

```diff
@@ -273,7 +273,8 @@ func (m *module) config() bool {
 func (m *module) addMember(member moduleMember) {
 	name := member.Name()
 	for _, existing := range m.members {
-		contract.Assertf(existing.Name() != name, "unexpected duplicate module member %s", name)
+		contract.Assertf(existing.Name() != name, "unexpected duplicate module member %q in %q",
+			name, m.name.String())
```

Comment:
So log as m.name.String() never panics (e.g. m.name == nil)  LGTM.

Reply:
It doesn't. `m` cannot be `nil` (or L275 would panic) and `m.name` is `type Module Token` (`Token` is `type Token string`). `Module.String()` is `func (tok Module) String() string { return string(tok) }`.